src/uiprotect/data/public_devices.py

```
@@ -131,6 +140,235 @@ class PublicLedSettings(ProtectBaseObject):
+class PublicLcdMessage(ProtectBaseObject):
+    # Spec marks ``type``/``text`` required, but the PATCH endpoint accepts
+    # (and a cleared message returns) ``{}`` — so every field is optional.
+    type: str | None = None
+    reset_at: int | None = None
+    text: str | None = None
+
+
+class PublicCameraFeatureFlags(ProtectBaseObject):
+    support_full_hd_snapshot: bool = False
+    has_hdr: bool = False
+    smart_detect_types: list[SmartDetectObjectType] = []
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)